pkg/tests/ossm-federation/testdata/traffic-splitting/east-mesh/smcp.yaml

```diff
@@ -89,6 +88,8 @@ spec:
                 memory: 128Mi
               limits: {}
   security:
+    identity:
+      type: ThirdParty
```

Comment:
Same here: ThirdParty is only when is executed in ROSA and traffic spliting test is for singles cluster I saw in the test name: TestSingleClusterFederationDifferentCerts